### DISCUSSION TITLE
can you use the Neo Store without F-droid?

### DISCUSSION THREAD

**Discussion by christopher-sal** [NeoApplications/Neo-Store#344] [Q&A] [answered]
F-droid takes up some space on my phone and I want to use Neo without F-droid

is that possible?

**Comment by ItamiWasTaken** [comment]
You can disable the fdroid repo and only use select ones

**Comment by ItsHarper** [comment, marked as answer]
Neo Store does not depend on having the official F-Droid app installed.

**Comment by christopher-sal** [comment]
well thanks :)
